.github/issue_template.md

```diff
@@ -1,12 +1,33 @@
+> Below you will find NVDA's Github issue template. You might consider swapping to the preview tab in order
+> to read through this template with the formatting applied before attempting to complete it. In all but
+> exceptional circumstances we require this to be completed. Your issue will likely be closed if this
+> template has not been followed. If you have trouble following this template, or with the initial investigation
+> that is required, please contact the fantastic community of people on the [NVDA users mailing list](https://github.com/nvaccess/nvda-community/wiki/Connect#international-users-mailing-list-english)
+> and politely ask for assistance.
+>
+> To complete the template please remove all lines that start with a greater than (>) symbol.
+> These lines are meant to provide an example of how you might complete this section of the issue template.
+>
+> **Please note:**
+> While we welcome images/screenshots to help explain a problem, be aware that many of the
+> developers of NVDA are blind and will greatly appreciate this image being described in text.
+> For example, to capture the output of the inspect.exe tool:
```

Comment:
I think it may be confusing to some why you are suddenly mentioning inspect here. Perhaps you should  say something more like: 
Rather than taking a screenshot showing the output of an accessibility tool, in many cases you can directly copy the text. For instance when using the Inspect tool from the Windows SDK, you can...